configuration: SCIP
data_file: data/mznc2019/amaze/2012-03-29.dzn
error: "586 POSTs [ 586,0,0,0,0,0,0,0,0,0, ], LINEQ [ 0,0,0,0,0,0,0,0,80, ], 160 /\
  \ 160 vars, 160 cliques, 2 / 7.55 / 11 NSubIntv m/a/m, 0 / 0.0298013 / 9 SubIntvSize\
  \ m/a/m, 0+156(156) clq eq_encoded  ... [var.c:5628] ERROR: cannot multi-aggregate\
  \ a fixed variable\n[scip_var.c:8536] ERROR: Error <-9> in function call\n[presol_milp.cpp:557]\
  \ ERROR: Error <-9> in function call\n[presol.c:462] ERROR: Error <-9> in function\
  \ call\n[scip_solve.c:936] ERROR: Error <-9> in function call\n[scip_solve.c:1213]\
  \ ERROR: Error <-9> in function call\n[scip_solve.c:1377] ERROR: Error <-9> in function\
  \ call\n[scip_solve.c:2398] ERROR: Error <-9> in function call\n[scip_solve.c:2620]\
  \ ERROR: Error <-9> in function call\n[/home/dekker1/half-reif-benchmarks/software/minizinc/solvers/MIP/MIP_scip_wrap.cpp:895]\
  \ ERROR: Error <-9> in function call\nSCIP Error (-9): method cannot be called with\
  \ this type of data\n  MIPScipWrapper runtime error, see output:  \nTERMINATING.\n\
  \  MIPScipWrapper runtime error, see output:"
evaluatedReifiedConstraints: 1478
flatIntConstraints: 1675
flatIntVars: 2040
flatTime: 0.265498
method: satisfy
model: data/mznc2019/amaze/amaze3.mzn
paths: 0
problem: amaze
status: ERROR
